stream-chat-android-client/src/test/java/io/getstream/chat/android/client/socket/HealthMonitorTest.kt

```diff
@@ -32,16 +38,32 @@ import org.mockito.kotlin.whenever
 
 internal class HealthMonitorTest {
 
+    companion object {
+        @JvmField
+        @RegisterExtension
+        val testCoroutines = TestCoroutineExtension()
+    }
+
+    @BeforeEach
+    fun setUp() {
+        StreamLog.setValidator { _, _ -> true }
+        StreamLog.setLogger(
+            KotlinStreamLogger(now = {
+                testCoroutines.dispatcher.scheduler.currentTime
+            })
+        )
+    }
+
```

Comment:
Can we create a helper method to init logger?
It seems to be repeated across our tests

Reply:
Makes sense 👌